src/transformers/pipelines/__init__.py

```diff
@@ -108,6 +108,14 @@
     "ner": "token-classification",
 }
 SUPPORTED_TASKS = {
+    "automatic-speech-recognition": {
+        "impl": AutomaticSpeechRecognitionPipeline,
+        "tf": (),
+        # Only load from `config.architectures`, AutoModelForCTC and AutoModelForConditionalGeneration
+        # do not exist yet.
+        "pt": () if is_torch_available() else (),
```

Comment:
Good for me!

Reply:
Guess we could/should change that to `AutoModelForCTC, AutoModelForConditionalGeneration` once `AutoModelForCTC` exisits

Reply:
Yes we definitely should !